### ISSUE TITLE
Please add support for "windows 8.1 with bing" .img files... if possible

### ISSUE THREAD

**Issue by pitsi** [ventoy/Ventoy#382]
Dear developer.

I used ventoy for the very first time yesterday. I copied sergei strelec's winpe iso and let ventoy simply boot it so as to do some checks on a very low end laptop (specs if requested) that came with windows 8.1 with bing preinstalled.
Then I thought about placing the img file of win8.1 with bing in it and let ventoy boot it instead of writing it on a new usb stick with rufus. However, the img file did not show up on the main manu of ventoy, quite possibly because it is not an iso.

If anyone is willing to test it, these are the names and the sums for the english img files that ms provides
```
File info: X19-57134_SW_DVD9_NTRL_Win_with_Bing_8.1_32BIT_English_OEM.img
Size: 2.77 GB
SHA1: 63422A198729941F47E7F3563197B1F3AC8C5640

File info: X19-57272_SW_DVD9_NTRL_Win_with_Bing_8.1_64BIT_English_OEM.img
Size: 3.71 GB
SHA1: BDC1E5FDBC1FAD5FE26155BAB6DAB7D95F434DEA
```

Thank you in advance.

**Comment by jendalinda** [comment]
Did you try to rename the files to ISO?

**Comment by pitsi** [comment]
Good idea! No, I will try it tomorrow... because I have to download them again and I will report back.

**Comment by pitsi** [comment]
It doesn't work. It shows up as an entry on ventoy's main menu but it does not boot at all, stopping at a black screen with a flashing cursor at the top left corner.

Ventoy's 1.19 changelog mentions
> Experimental support for booting IMG file (only two Linux distros supported now)

so that gives me some hope for support in the future.

**Comment by jendalinda** [comment]
I ve tested them too and they worked fine for me. No idea what's wrong.

**Comment by pitsi** [comment]
Are you on a uefi or a bios system? I tested it on a bios one... and I do not have a uefi one handy right now.

**Comment by jendalinda** [comment]
I have an UEFI system. I've tested both UEFI and Legacy (BIOS) modes. The 64 bit image works in both modes, the 32 bit image works only in Legacy mode.

**Comment by pitsi** [comment]
I finally tested it on a old laptop that had both bios and uefi. The img files did not start at all and ventoy showed a message saying they are not supported yet.
They did start as they should when I renamed them to iso and, as you mention, the 64bit image worked on both modes while the 32bit one worked only on bios mode.

**Comment by ventoy** [comment]
Windows .img file is not supported.
